### PR TITLE
more convienient queries with mynosql-query

### PR DESCRIPTION
**Pull request by dominictarr**
this adds a special case to the cli tool to use mynqsql-query to parse queries.
it makes it easy to query the data base with a one liner.

for example, get all messages where the first message (after init) is a post

``` bash
sbot query 'sequence=1, content.type=post'
```
will produce a lot of JSON output.

### CODE REVIEW

File: bin.js [hunk 1]
```diff
@@ -60,8 +61,11 @@ function usage () {
 var opts = require('minimist')(process.argv.slice(2))
 var cmd = opts._[0]
 var arg = opts._[1]
+
 delete opts._
 
+console.error(parse(arg))
```
Comment (pfraze) [inline bin.js]: is this debug?

File: bin.js [hunk 2]
```diff
@@ -157,6 +161,14 @@ function next (data) {
         throw explain(err, 'auth failed')
     }
 
+    console.log(cmd, arg, data)
```
Comment (pfraze) [inline bin.js]: same question

**Comment by dominictarr** [comment]
fixed

**Comment by pfraze** [comment]
ok cool, looks good to me

**Comment by dominictarr** [comment]
okay merging

**Comment by dominictarr** [comment]
merged into 4.2.0

**Pull request merged**